EIPS/eip-6596.md

```diff
@@ -0,0 +1,717 @@
+collections, and for "name", "descriptions", and “image” attributes to represent assets. Metadata plays a big role in
+facilitating the search and discovery of NFTs on marketplaces. As the number and variety of NFT projects increases,
+disconnected and limited metadata structures makes it difficult for collectors to navigate through the ocean of NFT
+assets on major marketplaces and make sense of their value.
+
+The provenance and history of artworks, artifacts and historical IP can take different forms and be created by
+different issuers. While there is no consolidated archive, a standardised metadata structure provides connectivity
+across NFTs associated with historical and cultural assets, regardless of the medium and issuer. For example, a video
+clip issued by a local filmmaker about the 1997 handover of Hong Kong can be connected to the news coverage by South
+China Morning Post (SCMP) of the
+Sino-British Joint Declaration via the HAT metadata. While well-informed collectors may not need this programmed
+connectivity, a standardised metadata for HATs will make these connections easily accessible for all collectors,
+scholars, and interested parties.
+
+Provenance and context are extremely important for historical assets. The archival and research of significant artworks,
+objects, and collections are ongoing; it is a collaborative effort between creators, artist estates, galleries, auction
+houses, scholars and institutions. The consolidation of information and knowledge is not only crucial to the
+understanding of our cultural heritage; the richer the context, the more valuable the asset is to collectors. A shared
+and standardised metadata structure enriches the context of an NFT. With the establishment of HATs, it is our hope that
+this standardised structure will contribute to the evolution of cultural heritage, and fully capture the context of
+culturally and historically significant assets, thereby ensuring relevant and fair value for all historically
+significant NFTs.
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT
+RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+This EIP extends the "Metadata JSON Schema" and "Metadata URI JSON Schema" schemas in [ERC-721](./eip-721.md) and
+[ERC-1155](./eip-1155.md) to have additional properties to capture the historical  significance of the underlying asset.
+
+Compatible contracts, in addition to implementing the
+relevant metadata schemas ("Metadata JSON Schema" for [ERC-721](./eip-721.md) contracts or "Metadata URI JSON Schema"
+for [ERC-1155](./eip-1155.md) contracts), must be extended to contain a `properties` property.
+
+### Historical Asset Metadata JSON Schemas
+
+For [ERC-721](./eip-721.md) smart contracts, the Metadata JSON Schema is:
```

Comment:
You shouldn't provide the full metadata when combined, only the extension

Reply:
This has been addressed in commit  e8c48f5.